### ISSUE TITLE
Koppeling activiteiten met PCSA google calendar

### ISSUE THREAD

**Issue by jpingen** [lesteenman/pcsa-drupal-theme#78]
Zou super chill zijn :)

**Comment by lesteenman** [comment]
Waarschijnlijk vrij makkelijk te doen met https://www.drupal.org/project/date_ical.

**Comment by Maharball1** [comment]
Ja, kan dit niet wellicht wat eerder worden ingevoerd? Wellicht dat je óf een pcsa agenda hebt, óf dat je je persoonlijk kan abboneren op die dingen?

**Comment by Maharball1** [comment]
@lesteenman Is wel een issue als we die plugin gebruiken:

> Seeking New Maintainer
Date iCal is seeking a new maintainer. I will not be able to upgrade Date iCal to work in Drupal 8, as my code shop does not use it. If you'd like to take over maintainership of Date iCal, please post a comment to this issue.

**Comment by lesteenman** [comment]
An sich niet direct een issue, maar wel de moeite waard om mee te nemen inderdaad. Volgens mij zijn er nog wel wat plugins, zal even kijken of ik er een kan vinden met D7 en D8 module.
